src/main/java/scm/address/logic/parser/AddScheduleCommandParser.java

```diff
@@ -46,8 +47,7 @@ public AddScheduleCommand parse(String userInput) throws ParseException {
                         PREFIX_DESCRIPTION, PREFIX_START_DATETIME, PREFIX_END_DATETIME);
 
         if (!arePrefixesPresent(argMultimap, PREFIX_TITLE,
-                PREFIX_DESCRIPTION, PREFIX_START_DATETIME, PREFIX_END_DATETIME)
-                || !argMultimap.getPreamble().isEmpty()) {
+                PREFIX_DESCRIPTION, PREFIX_START_DATETIME, PREFIX_END_DATETIME)) {
```

Comment:
Will a non-empty preamble be allowed in this case?

Reply:
Yes, commands like " add_schedule title/Meeting d/Project discussion start/2023-03-21 15:00 end/2023-03-21 16:00" and " add_schedule  title/Meeting d/Project discussion start/2023-03-21 15:00 end/2023-03-21 16:00" work as if "add_schedule title/Meeting d/Project discussion start/2023-03-21 15:00 end/2023-03-21 16:00" was inputted

Reply:
Thanks I added it back